### PR TITLE
git-revert: use HEAD in example

### LINKED ISSUE #6624: git-revert: @ sign vs HEAD
Minor nitpick: current `tldr git-revert` has

```
 - Revert the most recent commit:
   git revert {{@}}
```
If we err on the side of clarity / backwards compatibility (even as late as 2021: https://stackoverflow.com/questions/17910096/what-does-the-at-sign-symbol-character-mean-in-git), then we should just say `git revert HEAD`.

### PR DESCRIPTION
**Pull request by amcintosh**
As of git v1.8.5 `@` is an alias for HEAD, so for clarity changing the docs to simply use HEAD.

Closes #6624

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]
Thanks for your contribution.

**Review by CleanMachine1 — APPROVED** [review]
LGTM, let another review come before we merge 

cc @navarroaxel

File: pages/common/git-revert.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 - Revert the most recent commit:
 
-`git revert {{@}}`
+`git revert HEAD`
```
Comment (navarroaxel) [inline pages/common/git-revert.md:8]: I feel that `HEAD` should be tokenized
```suggestion
`git revert {{HEAD}}`
```
@CleanMachine1 ?

Comment (CleanMachine1) [reply pages/common/git-revert.md:8]: I agree :+1:

**Review by navarroaxel — APPROVED** [review]
Committed because it's a style fix. Thanks @amcintosh

**Pull request merged**